lib/components/formik/Select.js

```diff
@@ -43,6 +42,7 @@ const SelectField = ({
       error={meta.touched && meta.error}
       getOptionValue={getOptionValue || ((option) => option.value)}
       isMulti={!!isMulti}
+      {...rest}
```

Comment:
isn't `otherProps` a better name? or that's what we've followed throughout the project.